### PR TITLE
barys: Clarify invalid argument reason

### PR DESCRIPTION
**Pull request by agherzan**
Signed-off-by: Andrei Gherzan <andrei@resin.io>

### CODE REVIEW

File: build/barys [hunk 1]
```diff
@@ -220,15 +220,15 @@ while [[ $# -ge 1 ]]; do
         --shared-downloads)
             # Argument needs to be non-empty and absolute path
             if [ -z "$2" ] || [ "${2:0:1}" != "/" ]; then
-                log ERROR "\"$1\" argument is invalid."
+                log ERROR "\"$1\" argument is invalid. Argument needs to be non-empty and absolute path."
```
Comment (Page-) [inline build/barys]: I would reword this as `Argument needs to be a non-empty, absolute path` or as `Argument needs to be non-empty and an absolute path.`

Comment (agherzan) [inline build/barys]: Done @Page- . Thanks.
